src/pages/Auth/SignupPage/SignupPage.module.scss

```
@@ -0,0 +1,330 @@
+      height: 36px;
+    }
+
+    &::after {
+      position: absolute;
+      top: 11px;
+      right: 15px;
+      background: no-repeat center/17px 12px url("https://static.koreatech.in/assets/img/bus_dropdown.png");
+      width: 17px;
+      height: 17px;
+      content: "";
+    }
+
+    &--active {
+      background-color: #175c8e;
+      color: #fff;
+    }
+  }
+
+  &__content {
+    position: absolute;
+    z-index: 5;
+  }
+
+  &__option {
+    width: 192px;
+    height: 45px;
+    padding: 0 15px;
+    box-sizing: border-box;
+    background-color: #fff;
+    cursor: pointer;
+    font-size: 15px;
+    font-weight: 400;
+    line-height: 45px;
+    letter-spacing: -0.8px;
+    color: #252525;
+    border: 1px solid #d2dae2;
+
+    &:hover {
+      background-color: #175c8e;
+      color: #fff;
+    }
+
+    @include media.media-breakpoint(mobile) {
+      width: 155px;
+    }
+  }
+}
+
+.signup {
+  display: flex;
+  flex-wrap: wrap;
+
+  @include media.media-breakpoint(mobile) {
+    width: 320px;
+    margin: 0 auto;
+  }
+
+  &__advice {
+    display: block;
+    width: 100%;
+    height: 12px;
+    font-size: 12px;
+    font-weight: 300;
+    line-height: 1.5;
+    letter-spacing: -0.6px;
+    text-align: left;
+    color: #858585;
+    margin-bottom: 16px;
+    margin-left: 5px;
+
+    @include media.media-breakpoint(mobile) {
+      width: 320px;
+      margin: 0 auto 12px;
+    }
+  }
+
+  &__row {
+    display: flex;
+
+    &--nickname {
+      flex-wrap: nowrap;
+      width: 100%;
+
+      @include media.media-breakpoint(mobile) {
+        margin: 0 auto;
+        width: 320px;
+      }
+    }
+  }
+
+  &__button {
+    height: 45px;
+    background-color: #175c8e;
+    border: 1px solid #175c8e;
+    font-family: NanumSquare, sans-serif;
+    line-height: 1.3;
+    letter-spacing: -1px;
+    text-align: center;
+    color: #fff;
+    cursor: pointer;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    &:disabled {
+      background: #e5eaf0;
+      border: solid 1px #d2dae2;
+    }
+
+    &--nickname {
+      font-size: 12px;
+      width: 79px;
+      margin-left: 6px;
+
+      @include media.media-breakpoint(mobile) {
+        width: 75px;
+        height: 36px;
+      }
+    }
+
+    &--block {
+      width: 100%;
+    }
+
+    &--flex-end {
+      order: 99;
+    }
+
+    &--large-font {
+      font-size: 20px;
+      font-weight: bold;
+
+      @include media.media-breakpoint(mobile) {
+        font-size: 15px;
+      }
+    }
+  }
+
+  &__section {
+    text-align: left;
+    margin: 20px 0 6px;
+    border-top: 0.5px solid #d2dae2;
+    padding: 20px 0;
+    width: 100%;
+
+    @media (max-width: 576px) {
+      width: 320px;
+      margin: 20px auto 8px;
+    }
+
+    &--flex-end {
+      order: 99;
+    }
+  }
+
+  &__checkbox {
+    width: 16px;
+    height: 16px;
+    left: 0;
+    bottom: 1px;
+    background-color: #fff;
+    border: 1px solid #d2dae2;
+    line-height: 13px;
+
```

```suggestion
    appearance: none;
```
해당 속성이 추가되어야 기본 버튼을 벗어나서 디자인해둔 체크박스가 나올겁니다.